Orders.Logic/OrderManager.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Orders.Logic
+{
+    public class OrderManager
+    {
```

Comment:
OrderManager is not a good candidate for the Factory or Singleton pattern. It's scoped to incoming requests, therefore we will elect to keep it in this format

Reply:
We need to add an interface for OrderManager; useful for DI and allows it to be Mocked